pkgs/by-name/ke/keto/default.nix

```
@@ -0,0 +1,32 @@
+{ fetchFromGitHub, buildGoModule, lib, stdenv }:
+
+buildGoModule rec {
+  pname = "keto";
+  version = "0.13.0-alpha.0";
```

```suggestion
let
  pname = "keto";
  version = "0.13.0-alpha.0";
in
buildGoModule {
  inherit pname version;
```
Using `rec` is discouraged, please take a look at: https://nix.dev/guides/best-practices#recursive-attribute-set-rec